src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -1115,7 +1073,7 @@ public void testCreationWithErrorBadCaseFile() throws Exception {
             Message<byte[]> message = output.receive(1000);
             assertEquals("", new String(message.getPayload()));
             MessageHeaders headers = message.getHeaders();
-            assertEquals("newStudy", headers.get(StudyService.HEADER_STUDY_NAME));
+            //assertEquals("newStudy", headers.get(StudyService.HEADER_STUDY_NAME));
```

Comment:
delete dead code. Or are we not ready yet to remove the studyname from the message ?